### DISCUSSION TITLE
Why the new name?

### DISCUSSION THREAD

**Discussion by MagicAndre1981** [winsiderss/systeminformer#1260] [Questions / Answers] [answered]
Why does ProcessHacker has this new name? Looks like "processhacker" was acquired and this required rename? 🤔

**Comment by MagicAndre1981** [comment, marked as answer]
ok, found the answer here:

https://github.com/winsiderss/systeminformer/issues/1253

**Comment by Darthagnon** [comment]
If anyone wants the last version before the rebranding, Process Hacker v3.0.4953, [here's a mirror](https://github.com/ProcessHackerRepoTool/nightly-builds-mirror/releases/tag/3.0.4953)

**Reply by Darthagnon** [reply]
Official mirror's now gone; I posted the binaries in #1253
